examples/tilted_bottom_boundary_layer.jl

```diff
@@ -134,6 +132,14 @@ drag_bc_v = FluxBoundaryCondition(drag_v, field_dependencies=(:u, :v), parameter
 u_bcs = FieldBoundaryConditions(bottom = drag_bc_u)
 v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
 
+# Note that, similar to the buoyancy boundary conditions, we had to
+# include the background flow in the drag calculation.
+#
+# Let us also create `BackgroundField` for the along-slope interior velocity:
+
+@inline constant_velocity(x, z, t, p) = p.V∞
+V∞_field = BackgroundField(constant_velocity, parameters=(; V∞))
```

Comment:
```suggestion
V∞_field = BackgroundField(ConstantField(V∞))
```

It's also easy to make `BackgroundField(V∞)` work. I'm surprised we haven't added that!

Reply:
It's just a matter of adding

```julia
regularize_background_field(LX, LY, LZ, c::Number, grid, clock) = ConstantField(c)
```

https://github.com/CliMA/Oceananigans.jl/blob/main/src/Fields/background_fields.jl#L63

then you can write

```julia
V∞_field = BackgroundField(V∞)
```

Reply:
Definitely don't want to encourage the rather convoluted approach of writing a function to return a constant! We must be a force for good

Reply:
> It's also easy to make BackgroundField(V∞) work. I'm surprised we haven't added that!

Yeah I added that at first and was also surprised that there was no method for that.

I think I'll add it in another single-purpose PR and then do it more cleanly here :)

Reply:
I can add it if you want, it'll be less work

Reply:
The CI takes annoyingly long to finish so its nice to combine steps like this I think

Reply:
Ah, sorry I may have jumped the gun!

Reply:
no worries.

I usually feel swamped so try to do things like this quick. But it's good to take care.